src/CookieV3.php

```diff
@@ -34,7 +34,7 @@ protected function isCookieFormatValid($cookie) {
     }
 
     public function getBlockAction() {
-        return $this->getDecodedCookie()->a == 'c' ? 'captcha' : 'block';
+        return $this->getDecodedCookie()->a;
```

Comment:
I don't see how this implementation holds - the decoded cookie 'a' is a letter and based on the method